src/vs/workbench/browser/actions/quickAccessActions.ts

```diff
@@ -147,10 +150,32 @@ registerAction2(class QuickAccessAction extends Action2 {
 
 	run(accessor: ServicesAccessor, prefix: undefined): void {
 		const quickInputService = accessor.get(IQuickInputService);
-		quickInputService.quickAccess.show(typeof prefix === 'string' ? prefix : undefined, { preserveValue: typeof prefix === 'string' /* preserve as is if provided */ });
+		const searchText = getFileSearchText(accessor) ?? '';
+		quickInputService.quickAccess.show(typeof prefix === 'string' ? prefix : searchText, { preserveValue: typeof prefix === 'string' /* preserve as is if provided */ });
 	}
 });
 
+function getFileSearchText(accessor: ServicesAccessor): string | null {
+	const editorService = accessor.get(IEditorService);
+	const configurationService = accessor.get(IConfigurationService);
+
+	const activeEditor: IEditor = editorService.activeTextEditorControl as IEditor;
+	if (!activeEditor) {
+		return null;
+	}
+	if (!activeEditor.hasTextFocus()) {
+		return null;
+	}
+
+	// only happen if it would also happen for the search view
+	const seedSearchStringFromSelection = configurationService.getValue<boolean>('editor.find.seedSearchStringFromSelection');
```

Comment:
We should have our own setting. Follow what we did for `preserveInput`

Reply:
Got it, added that! Great suggestion @TylerLeonhardt . Let me know if that's correct! Thanks